ludwig/encoders/date_encoders.py

```diff
@@ -15,21 +15,25 @@
 # limitations under the License.
 # ==============================================================================
 import logging
-import math
 from abc import ABC
+from typing import Dict, List, Optional
 
-# import tensorflow as tf
+import torch
 
 from ludwig.encoders.base import Encoder
+from ludwig.utils import torch_utils
 from ludwig.utils.registry import Registry, register
 from ludwig.modules.embedding_modules import Embed
 from ludwig.modules.fully_connected_modules import FCStack
 
 logger = logging.getLogger(__name__)
 
-
 ENCODER_REGISTRY = Registry()
 
+# Year, month, day, weekday, yearday, hour, minute, seconds, second_of_day.
+# TODO: Share this constant with date_feature.DATE_VECTOR_SIZE.
```

Comment:
nit: You could import `DATE_INPUT_SIZE` in `date_feature.py` and remove the code that sets `DATE_VECTOR_LENGTH`.

So the `input_shape` property of `DateInputFeature` could return `torch.Size([DATE_INPUT_SIZE])`.

Reply:
That would work. Another idea would be to have a constants.py file and put all of the constants there. I'll think about this more, and send a PR for this if it seems good.